### PR TITLE
Trips

### PR DESCRIPTION
**Pull request by addisonnick**
Updated. Add/delete trips from myTrips with their id's. When you get trips, it will return the entire trip object now.

### CODE REVIEW

**Comment by ToucheSir** [comment]
:+1: 

Quick question: what is the expected format/structure/schema is the body (`updateMyTrips`) supposed to have?

File: routes/accounts.js [hunk 1]
```diff
@@ -153,6 +161,43 @@ module.exports = (db, auth) => {
...
+              } else {
+                Promise.all(docs.map(pinInfoForTrip))
+                  .then(trips => res.status(200).json(trips));
+              }
+            });
+        }
+      });
+
+  //update myTrips to add or delete trip id's
+  }).put((req, res) => {
```
Comment (ToucheSir) [inline routes/accounts.js]: If there's no trip id here, could this be a `POST` instead?

**Comment by addisonnick** [comment]
This should be more straightforward. To add trip to myTrips, call post, accounts/:id/mytrips with body format: { "myTrips": id }. Very similar for delete

**Comment by ToucheSir** [comment]
@addisonnick Are we good to go? If so, let's :shipit:!

**Pull request merged**